src/Management/ResourceBuilder.php

```diff
@@ -270,8 +272,8 @@ private function buildContentTypeField(array $data): Field\FieldInterface
         $hydratorData = [
             'id' => $data['id'],
             'name' => $data['name'],
-            'required' => $data['required'],
-            'localized' => $data['localized'],
+            'required' => $data['required'] ?? null,
+            'localized' => $data['localized'] ?? null,
```

Comment:
Yes, as you can see in the first response body of `e2e_snapshot_content_type_get_collection.json`, the field `description` of the content type is returned without those fields